src/web/routes/guidance.js

```diff
@@ -0,0 +1,71 @@
+const { compose, equals, prop } = require('ramda')
+const { handleRequestForPath } = require('./application/middleware')
+
+const pages = [
+  {
+    title: 'How it works',
+    path: '/how-it-works'
+  },
+  {
+    title: 'Eligibility',
+    path: '/eligibility'
+  },
+  {
+    title: 'What you can buy',
+    path: '/what-you-can-buy'
+  },
+  {
+    title: 'Using your card',
+    path: '/using-your-card'
+  },
+  {
+    title: 'Apply for Healthy Start',
+    path: '/apply-for-healthy-start'
+  },
+  {
+    title: 'Report a change',
+    path: '/report-a-change'
+  }
+]
+
+const isPathForPage = (path) => compose(equals(path), prop('path'))
```

Comment:
I don't think this method name is quite right, how about something like `hasMatchingPath`?